### PR TITLE
Bug 1805836 - Hide warning when ETP is enabled and show it when disabled

### PR DESCRIPTION
**Pull request by mcarare**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation

Used by GitHub Actions.

### CODE REVIEW

**Review by Mugurell — APPROVED** [review]

**Comment by mcarare** [comment]
@Mugurell can you please re-review this, with the added commit? TY!

File: focus-android/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt [hunk 1]
```diff
@@ -670,9 +670,9 @@ class BrowserFragment :
         val requestDesktop = tab.content.desktopMode
 
         val addToHomescreenDialogFragment = AddToHomescreenDialogFragment.newInstance(
-            tab.content.url,
-            tab.content.titleOrDomain,
-            tab.trackingProtection.enabled,
+            url = tab.content.url,
+            title = tab.content.titleOrDomain,
+            blockingEnabled = !tab.trackingProtection.ignoredOnTrackingProtection,
```
Comment (Mugurell) [inline focus-android/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt:675]: This change matches another similar usecase - https://searchfox.org/mozilla-mobile/rev/3efcbbbe98c6bde158c0ea6df9326aef211f6bf8/firefox-android/focus-android/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt#887 so LGTM.

**Comment by mcarare** [comment]
@Mergifyio backport releases_v109.0

**Comment by mcarare** [comment]
@Mergifyio backport releases_v109

**Pull request merged**